### PR TITLE
Modal Popup

### PR DESCRIPTION
**Pull request by adel-gu**
### In this PR we:
- Setup linter for `HTML` & `CSS` & `JS` project.
- Generate project modal using JavaScript.

### CODE REVIEW

File: scripts/modal.js [hunk 1]
```diff
@@ -0,0 +1,223 @@
+const project1 = {
```
Comment (SajjadAhmad14) [inline scripts/modal.js:1]: Please align the project title and close button vertically on the mobile view. Kindly see the screenshot of your project below.


popup-mobile

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Adel, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏
- [x] Please add all the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter configuration for html+css+javaScript.
- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: scripts/modal.js (hunk 1 above)
Comment (adel-gu) [reply scripts/modal.js:1]: Hi **@SajjadAhmad14**  👋,

Thank you so much for the highlights  👏 however, am having a hard time comprehending your comment on the alignment of the project title with the close button.

I actually followed the design as much as I can, so kindly asking can you please explain it more? 

### Figma Design 
image

### My Design
image

and once again **thank you very much**.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @adel-gu 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the changes requested by the previous reviewer.
- Great communication. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**